cmake/catkin_package.cmake

```diff
@@ -374,6 +377,15 @@ function(_catkin_package)
       endif()
     endforeach()
 
+    set(PKG_COMPILE_OPTIONS ${${PROJECT_NAME}_COMPILE_OPTIONS})
+    separate_arguments(PKG_COMPILE_OPTIONS)
```

Comment:
Same as for the next line. Why should we allow the user to specify a space separated string if CMake provides support for lists natively?